apps/docs/app/globals.css

```
@@ -1,26 +1,259 @@
-@import "tailwindcss";
+@tailwind base;
```

Stylelint (17.10.0)

[error] Unexpected unknown at-rule "@tailwind" (scss/at-rule-no-unknown)

(scss/at-rule-no-unknown)